hybrasyl/Scripting/HybrasylUser.cs

```diff
@@ -302,10 +303,25 @@ public bool GiveItem(string name, int count = 1)
                 return true;
             }
             return false;
-        }    
+        }
 
         public bool TakeItem(string name)
         {
+            if (User.Inventory.ContainsName(name))
+            {
+                // Find the first instance of the specified item and remove it
+                var slots = User.Inventory.SlotByName(name);
+                if (slots.Length == 0)
+                {
+                    GameLog.ScriptingWarning("{Function}: User had {item} a moment ago but now it's gone...?",
```

Comment:
Can we move string values to the localization array?

Reply:
This is a log line, and is not displayed to a client. If you are saying that log lines need to be localized, that is a good job for someone who wants to provide translations for the lines and use the server in a different primary language.